AISKU/src/Snippet.ts

```diff
@@ -15,4 +15,5 @@ export interface Snippet {
     queue?: Array<() => void>;
     sv?: string;
     version?: number;
+    cr?: boolean; // cdn retry would be proceed if ture
```

Comment:
Once we have tested this, we will (I think) want this to be the default, so then if someone doesn't want to retry they would be required to set this to `false` and then we just change `if (appInsights.cr){` to `if (appInsights.cr !== false){` so that anything other then `false` would cause the retry to occur.